### PR TITLE
mew qr code

### PR DESCRIPTION
**Pull request by gesseekur**
REMINDER TO SELF: CHANGE MEW BUILDS LINK BEFORE MERGING 

### Feature

- [x] Updated CHANGELOG.md
- [x] Add PR label
- [x] No hard-coded strings

### CODE REVIEW

File: src/layouts/CreateWalletLayout/components/ScanToRedirectQR/ScanToRedirectQR.vue [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    getMobileOperatingSystem() {
+      const userAgent = navigator.userAgent || navigator.vendor || window.opera;
+
+      if (/windows phone/i.test(userAgent)) {
+        return;
+      }
+
+      if (/android/i.test(userAgent)) {
+        window.location.href =
+          'http://play.google.com/store/apps/details?id=PACKAGEURL';
```
Comment (gamalielhere) [inline src/layouts/CreateWalletLayout/components/ScanToRedirectQR/ScanToRedirectQR.vue]: Missing playstore id

File: src/layouts/CreateWalletLayout/CreateWalletLayout.vue [hunk 2]
```diff
@@ -75,6 +75,14 @@
                     </a>
                   </div>
                 </div>
+                <div class="qr-container mt-5">
+                  <qrcode
+                    :value="
+                      'https://www.mewbuilds.com/build/7924905fef21b6ad252cf5d91ecfb7b0d2a5ad81/index.html#/qr-code'
```
Comment (kvhnuke) [inline src/layouts/CreateWalletLayout/CreateWalletLayout.vue]: dont forget to switch the mewbuilds link :)

File: src/layouts/CreateWalletLayout/components/ScanToRedirectQR/ScanToRedirectQR.vue [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        window.location.href = 'https://itunes.apple.com/app/id1464614025';
+      }
+
+      if (
+        userAgent.match(
+          /SAMSUNG|Samsung|SGH-[I|N|T]|GT-[I|N]|SM-[A|N|P|T|Z]|SHV-E|SCH-[I|J|R|S]|SPH-L/i
+        )
+      ) {
+        window.location.href =
+          'https://galaxystore.samsung.com/detail/com.myetherwallet.mewwallet?session_id=W_29942474fffbd49127ee223d697f1518';
```
Comment (kvhnuke) [inline src/layouts/CreateWalletLayout/components/ScanToRedirectQR/ScanToRedirectQR.vue]: link is just`https://galaxystore.samsung.com/detail/com.myetherwallet.mewwallet`

File: src/layouts/CreateWalletLayout/CreateWalletLayout.vue [hunk 4]
```diff
@@ -75,6 +75,14 @@
                     </a>
                   </div>
                 </div>
+                <div class="qr-container mt-5">
+                  <qrcode
+                    :value="
+                      'https://www.mewbuilds.com/build/c957ff1995b4e3d7aca9c49370f74628810e1a30/index.html#/qr-code'
```
Comment (kvhnuke) [inline src/layouts/CreateWalletLayout/CreateWalletLayout.vue]: this should be `https://www.myetherwallet.com/qr-code`

**Pull request merged**